extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/DatabaseCommands.scala

```diff
@@ -68,38 +69,50 @@ object DatabaseCommands {
   val AnalyzeTables = {
     DatabaseCommandSpec(
       "org.apache.spark.sql.execution.command.AnalyzeTablesCommand",
-      Seq(DatabaseDesc("databaseName", "StringOptionDatabaseExtractor", true)),
+      Seq(DatabaseDesc("databaseName", "StringOptionDatabaseExtractor", isInput = true)),
       "ANALYZE_TABLE")
   }
 
   val SetDatabase = {
     val cmd = "org.apache.spark.sql.execution.command.SetDatabaseCommand"
-    val databaseDesc = DatabaseDesc("databaseName", "StringDatabaseExtractor", true)
+    val databaseDesc = DatabaseDesc("databaseName", "StringDatabaseExtractor", isInput = true)
     DatabaseCommandSpec(cmd, Seq(databaseDesc), "SWITCHDATABASE")
   }
 
   val DescribeDatabase = {
     val cmd = "org.apache.spark.sql.execution.command.DescribeDatabaseCommand"
-    val databaseDesc = DatabaseDesc("databaseName", "StringDatabaseExtractor", true)
+    val databaseDesc = DatabaseDesc("databaseName", "StringDatabaseExtractor", isInput = true)
     DatabaseCommandSpec(cmd, Seq(databaseDesc), "DESCDATABASE")
   }
 
   val SetCatalogAndNamespace = {
     val cmd = "org.apache.spark.sql.catalyst.plans.logical.SetCatalogAndNamespace"
-    val databaseDesc1 = DatabaseDesc("child", "ResolvedDBObjectNameDatabaseExtractor", true)
-    val databaseDesc2 = DatabaseDesc("namespace", "StringSeqOptionDatabaseExtractor", true)
+    val databaseDesc1 =
+      DatabaseDesc("child", "ResolvedDBObjectNameDatabaseExtractor", isInput = true)
+    val databaseDesc2 =
+      DatabaseDesc(
+        "namespace",
+        "StringSeqOptionDatabaseExtractor",
+        catalogDesc = Some(CatalogDesc(
+          fieldName = "catalogName",
+          fieldExtractor = "StringOptionCatalogExtractor")),
+        isInput = true)
     DatabaseCommandSpec(cmd, Seq(databaseDesc1, databaseDesc2), "SWITCHDATABASE")
   }
 
   val SetNamespace = {
     val cmd = "org.apache.spark.sql.execution.command.SetNamespaceCommand"
-    val databaseDesc = DatabaseDesc("namespace", "StringSeqDatabaseExtractor", true)
+    val databaseDesc = DatabaseDesc(
```

Comment:
irrelevant changes

Reply:
No logic changed. Code change here is adapting to new field `catalogDesc` added before `isInput` of `DatabaseDesc`, same way as `TableDesc`.